bucket/archwsl.json

```diff
@@ -43,30 +43,19 @@
         "    $installable = $false",
         "}",
         "if ($installable) {",
-        "    \"`$si = New-Object System.Diagnostics.ProcessStartInfo '$dir\\Arch.exe' -Property @{ UseShellExecute = `$false; RedirectStandardInput = `$true }",
-        "    `$p = [System.Diagnostics.Process]::Start(`$si)",
-        "    `$p.StandardInput.Write('y')",
-        "    `$p.WaitForExit()",
-        "    `$lx_uid = & '$dir\\Arch.exe' get --lxuid",
-        "    if(!`$lx_uid) { error 'ArchWsl is not installed!'; return }",
-        "    Set-ItemProperty -Path ('HKCU:Software\\Microsoft\\Windows\\CurrentVersion\\Lxss\\' + `$lx_uid) -Name BasePath -Value '$persist_dir\\data'",
-        "    if (Test-Path ('$persist_dir' + '\\data\\rootfs')) {",
-        "        Remove-Item -Recurse -Path '$dir\\rootfs'",
-        "    } else {",
-        "        Move-Item -Path '$dir\\rootfs' -Destination '$persist_dir\\data\\rootfs'",
-        "    }",
-        "    Remove-Item -Recurse -Path '$dir\\temp'",
+        "    \"Copy-Item `\"$dir\\Arch.exe`\" `\"$persist_dir\\data\\Arch.exe`\"",
+        "    `& '$persist_dir\\data\\Arch.exe' install `\"$dir\\rootfs.tar.gz`\";`$res=`$?",
+        "    Remove-Item ('$persist_dir'+'\\data\\Arch.exe') -Force",
+        "    if(!`$res) { error 'ArchWsl installation failed!'; return }",
         "    \" | Set-Content \"$dir\\install.ps1\"",
         "    & \"$dir\\install.ps1\"",
```

Comment:
Why are you creating install file and then executing it instead of just running chained commands?

Reply:
This is because rootfs is placed in the executable file directory according to the API specification.

Reply:
What?

What is the difference when you run this in sequence and you put it in file and then execute that script?

```powershell
Copy-Item "$dir\Arch.exe" "$persist_dir\data\Arch.exe"
& "$persist_dir\data\Arch.exe" install "$dir\rootfs.tar.gz"
$res=$?
Remove-Item "$persist_dir\data\Arch.exe" -Force
if(!$res) { error 'ArchWsl installation failed!' }
```

Reply:
It may no longer be needed. 
The file was running with admin privileges but now it's running in user mode. (https://github.com/lukesampson/scoop-extras/pull/1877#issuecomment-476328832)

Reply:
I understood.
I'm sorry for my poor English

Reply:
I pushed the commit that changed the point you pointed out now.
I'm sorry I'm too late